src/Chat/Bot/Calculator.hs

```diff
@@ -25,15 +40,15 @@ calculate op a b =
 
 
 
--- LEVEL: Easy
+-- | LEVEL: Easy
```

Comment:
> Hehe, now I know a innocent bar actually is important 

One of the (many) reasons why doctests are silly. I am/was sorely tempted just to make them normal tests.